.circleci/config.yml

```diff
@@ -360,7 +360,7 @@ jobs:
           name: Build and publish falco-driver-loader-dev
           command: |
             cd /source/falco
-            docker buildx build --push --build-arg FALCO_IMAGE_TAG=master \
+            docker buildx build --push --build-arg FALCO_IMAGE_TAG=x86_64-master \
```

Comment:
We shall use just-pushed tag from previous `Build and publish falco-dev` job for current arch.

Same for other modified lines.
Otherwise, using master, we are using the latest merged PR falco:master image (because current falco:master image will be pushed by the `publish-docker-dev` job later.